python/ray/serve/_private/router.py

```diff
@@ -351,6 +380,30 @@ def target_num_scheduling_tasks(self) -> int:
     def curr_replicas(self) -> Dict[str, ReplicaWrapper]:
         return self._replicas
 
+    def _get_actor_name(self) -> str:
+        """Gets the name of the actor where this scheduler runs.
```

Comment:
router can run in a driver, in which case we shouldn't log a noisy exception when this fails

Reply:
Sounds good, I rewrote the logic to directly return an empty string when running in the driver.